patroni/ctl.py

```diff
@@ -247,10 +247,12 @@ def get_cursor(cluster, connect_parameters, role='master', member=None):
     return None
 
 
-def get_members(cluster, cluster_name, member_names, role, force, action, scheduled_at=None):
+def get_members(cluster, cluster_name, member_names, role, force, action):
```

Comment:
Let's add one more argument here, `ask_confirmation=True`.
It allows us to make conditional call of the `confirm_members_action()` directly from `get_members()` in most of the cases (only `restart()` will call it explicitly).

Reply:
done